frontend/src/components/campAdmin/CampSharingSettings.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <content-group
+    :title="$tc('components.campAdmin.campSharingSettings.title')"
+    icon="mdi-share-variant"
+  >
+    <v-skeleton-loader v-if="camp._meta.loading" type="article" />
+    <v-list class="py-0" color="transparent">
+      <v-list-item class="px-0">
+        <v-list-item-content>
+          <v-list-item-title>
+            {{ $tc(`components.campAdmin.campSharingSettings.${sharingStatus}.title`) }}
+          </v-list-item-title>
+          <i18n
+            :path="`components.campAdmin.campSharingSettings.${sharingStatus}.description`"
+            tag="div"
+            class="body-2 grey--text text--darken-3"
+          >
+            <template #team>
+              <span v-if="isOutsider">{{
+                $tc('components.campAdmin.campSharingSettings.team')
+              }}</span>
+              <router-link v-else :to="teamRoute">{{
+                $tc('components.campAdmin.campSharingSettings.team')
+              }}</router-link>
+            </template>
+          </i18n>
+          <v-list-item-subtitle v-if="camp.isShared" class="pb-1">{{
+            $tc('components.campAdmin.campSharingSettings.sharedSince', 1, {
+              sharedSince: dateTimeLong(camp.sharedSince),
+              sharedBy: userDisplayName(camp.sharedBy()),
+            })
+          }}</v-list-item-subtitle>
+        </v-list-item-content>
+        <v-list-item-action v-if="camp.isShared" class="align-self-start">
+          <v-tooltip top>
+            <template #activator="{ on, attrs }">
+              <v-btn
+                icon
+                v-bind="attrs"
+                small
+                v-on="on"
+                @click="copyCampUrlToClipboard()"
+              >
+                <v-icon>mdi-clipboard-check-multiple-outline</v-icon>
+              </v-btn>
+            </template>
+            {{ $tc(`components.campAdmin.campSharingSettings.copyCampLink`) }}
+          </v-tooltip>
+        </v-list-item-action>
+        <v-list-item-action v-if="isManager" class="align-self-start">
```

Comment:
As with all camp settings, the frontend only allows managers to change this, but in the API it is also possible for members. #3440